sdk/RELEASE.md

```diff
@@ -30,6 +30,7 @@
 * Depends on `google-auth>=1.6.1,<3` [\#6939](https://github.com/kubeflow/pipelines/pull/6939)
 * Change otherwise to else in yaml [\#6952](https://github.com/kubeflow/pipelines/pull/6952)
 * Avoid pydantic bug on Union type [\#6957](https://github.com/kubeflow/pipelines/pull/6957)
+* Improve CLI experience for archiving experiments, managing recurring runs and listing resources [\#6934](https://github.com/kubeflow/pipelines/pull/6934)
```

Comment:
Maybe move this up to the `## Major Features and Improvements` section?

Besides, can you help cherry pick this change into `sdk/release-1.8` branch once it's merged?

Reply:
Sounds good, moved the line item up to Major improvements (and rebased against `master`). 

Happy to help cherry-pick to `sdk/release-1.8` (this will be my first time). Guessing I need need to cherry-pick the resulting commit on `master` once this is merged?

Reply:
Yes, basically you'll create a brach off `sdk/release-1.8`, cherry pick the commit from `master` into your branch and send a pull request to merge to `sdk/release-1.8`.

Reply:
Cool I think I've got that done, see https://github.com/kubeflow/pipelines/pull/6991. Let me know if you need any tweaks, thanks for all the help!